.storybook/withThemeProvider.js

```diff
@@ -12,15 +12,15 @@ const themes = [noTheme, sageTheme].reduce((themesObject, theme) => {
   return themesObject;
 }, {});
 
-if (process.env.STORYBOOK_DEBUG_THEME === "true") {
+if (!isChromatic() && process.env.STORYBOOK_DEBUG_THEME === "true") {
```

Comment:
Due to placing the variables in `.env` I've had to add this to prevent it including the debug theme in the comparisons in chromatic. I assume it's because it uses it's own `build-storybook` script instead of ours and pulls in both variables in `.env`